codegen/cmd/injection-gen/generators/filtered_informer.go

```diff
@@ -129,6 +179,20 @@ func withInformer(ctx {{.contextContext|raw}}) ({{.contextContext|raw}}, []{{.co
 	return ctx, infs
 }
 
+func withDynamicInformer(ctx {{.contextContext|raw}}) {{.contextContext|raw}} {
+	untyped := ctx.Value({{.factoryLabelKey|raw}}{})
+	if untyped == nil {
+		{{.loggingFromContext|raw}}(ctx).Panic(
+			"Unable to fetch labelkey from context.")
+	}
+	labelSelectors := untyped.([]string)
+	for _, selector := range labelSelectors {
+		inf := &wrapper{client: {{ .clientGet|raw }}(ctx), selector: selector}
+		ctx = context.WithValue(ctx, Key{Selector: selector}, inf)
```

Comment:
you will want to register this usage of `context.WithValue` in the `m` map to prevent rando context imports.

Reply:
Done